evidence/data_sources/cancer_hotspots.py

```diff
@@ -3,40 +3,60 @@
 Downloads Site: https://www.cancerhotspots.org/#/download
 Data URL: https://www.cancerhotspots.org/files/hotspots_v2.xls
 """
+from os import remove
+import shutil
+from pathlib import Path
 from typing import Dict, Optional
 
 import pandas as pd
 import requests
 from variation.query import QueryHandler
+import boto3
+from botocore.config import Config
 
-from evidence import DATA_ROOT, logger, ENV_NAME
+from evidence import DATA_DIR_PATH, logger
 from evidence.schemas import SourceMeta, Response, Sources
 
 
 class CancerHotspots:
     """Class for Cancer Hotspots Data Access."""
 
     def __init__(
-        self, data_url: str = "https://www.cancerhotspots.org/files/hotspots_v2.xls"
+        self, data_url: str = "https://www.cancerhotspots.org/files/hotspots_v2.xls",
+        src_dir_path: Path = Path(DATA_DIR_PATH / "cancer_hotspots"),
+        normalized_data_path: Optional[Path] = None
     ) -> None:
         """Initialize Cancer Hotspots class
 
         :param str data_url: URL to data file
+        :param Path src_dir_path: Path to cancer hotspots data directory
+        :param Optional[Path] normalized_data_path: Path to normalized cancer
+            hotspots file
         """
         self.data_url = data_url
         fn = self.data_url.split("/")[-1]
-        self.data_path = DATA_ROOT / fn
-        self.normalized_data_path = DATA_ROOT / f"normalized_{fn}"
+        self.src_dir_path = src_dir_path
+        self.src_dir_path.mkdir(exist_ok=True, parents=True)
+        self.data_path = self.src_dir_path / fn
+        self.normalized_data_path = None
+        if not normalized_data_path:
+            self.get_normalized_data_path()
+        else:
+            if normalized_data_path.exists():
+                self.normalized_data_path = normalized_data_path
+            else:
+                logger.error(f"Normalized Cancer Hotspots path does not exist: "
```

Comment:
Something a bit clearer like "Normalized Cancer Hotspots data does not exist at provided path: " would help here imo